package.json

```diff
@@ -65,8 +67,10 @@
     "@mui/material": "^5.0.0",
     "@mui/styles": "^5.0.0",
     "clsx": "^1.1.1",
-    "prop-types": "^15.7.2",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "prop-types": "^15.7.2"
+  },
+  "peerDependencies": {
+    "react": ">=17.0.0",
+    "react-dom": ">=17.0.0"
```

Comment:
What's the advantage for this specific case?

Reply:
It conflicts with my branch since mine uses react 18. Moving it to peer deps, we can also bundle it with react 18.